variables.tf

```diff
@@ -78,3 +78,28 @@ variable "worker_sg_ingress_from_port" {
   description = "Minimum port number from which pods will accept communication. Must be changed to a lower value if some pods in your cluster will expose a port lower than 1025 (e.g. 22, 80, or 443)."
   default     = "1025"
 }
+
+variable "kubeconfig_context_name" {
+  description = "Name of the kubeconfig context."
+  default     = "aws"
```

Comment:
I think this should default to the cluster name as `aws` is very uninformative.

Reply:
Yea, I do agree (I configured my context name to be the cluster name as well). I was trying to limit the amount of changes made by default but if others think that is a more reasonable default I would be happy to provide another PR to fix it.

Reply:
Also, I just found that if you are trying to merge various EKS kubeconfigs together via the `KUBECONFIG` environment variable the username needs to be unique across all of the config files. So, I will go ahead and make the change for both the context name and the user name.

Reply:
That'd all be appreciated @sdavids13 . Thanks for driving that